include/GILGen/Context.hpp

```diff
@@ -183,6 +183,20 @@ class Context {
         return insertTerminator(new (_arena)
                                     gil::CondBrInst(cond, thenBB, elseBB));
     }
+
+    gil::CallInst *
+    buildCall(std::string const &opName, llvm::ArrayRef<gil::Value> args)
+    {
+        // Create a Function* with the operator name (prefixed with @)
+        auto *func = new (_arena) gil::Function(opName, nullptr);
```

Comment:
Well the call shouldn't create an empty function, it should retrieve an existing function
Sema should've found the correct function overload, just like RefExpr, the operator node should contain a link to the function declaration